ui/raidboss/data/06-ew/dungeon/another_sildihn_subterrane.txt

```diff
@@ -0,0 +1,225 @@
+123.1 "Puff and Tumble" duration 0.7 #sync / 1[56]:[^:]*:Silken Puff:7771:/
+129.5 "Soaping Spree" sync / 1[56]:[^:]*:Silkie:7767:/
+129.5 "Fizzling Duster x2" #sync / 1[56]:[^:]*:Silken Puff:776A:/
+129.5 "Chilling Duster x2" #sync / 1[56]:[^:]*:Silken Puff:7768:/
+134.6 "Bracing Suds / Fizzling Suds" sync / 1[56]:[^:]*:Silkie:(7757|7759):/ 
+136.7 "--sync--" sync / 1[56]:[^:]*:Silkie:79FB:/
+141.7 "Slippery Soap" sync / 1[56]:[^:]*:Silkie:775E:/
+145.7 "Bracing Duster / Fizzling Duster" sync / 1[56]:[^:]*:Silkie:(7764|7765):/
+146.0 "Soapsud Static?" #sync / 1[56]:[^:]*:Silkie:77ED:/
+160.1 "Total Wash" sync / 1[56]:[^:]*:Silkie:7750:/
+166.2 "Fresh Puff" sync / 1[56]:[^:]*:Silkie:7766:/
+172.3 "Eastern Ewers" sync / 1[56]:[^:]*:Silkie:776D:/
+178.6 "Rinse" sync / 1[56]:[^:]*:Eastern Ewer:776F:/ duration 7.3
+188.1 "Soaping Spree" sync / 1[56]:[^:]*:Silkie:7767:/
+188.1 "Bracing Duster / Chilling Duster / Fizzling Duster" #sync / 1[56]:[^:]*:Silken Puff:(7769|7768|776A):/
+188.1 "Fizzling Duster" #sync / 1[56]:[^:]*:Silken Puff:776A:/
+195.2 "Chilling Suds" sync / 1[56]:[^:]*:Silkie:7758:/
+202.3 "Carpet Beater" sync / 1[56]:[^:]*:Silkie:774F:/
+204.4 "--sync--" sync / 1[56]:[^:]*:Eastern Ewer:79FB:/
+209.4 "Slippery Soap" sync / 1[56]:[^:]*:Silkie:775E:/
+213.4 "Chilling Duster" sync / 1[56]:[^:]*:Silkie:7763:/
+225.9 "Bracing Suds" sync / 1[56]:[^:]*:Silkie:7757:/
+233.0 "Soaping Spree" sync / 1[56]:[^:]*:Silkie:7767:/
+233.0 "Chilling Duster?" #sync / 1[56]:[^:]*:Silken Puff:7768:/
+233.0 "Bracing Duster?" #sync / 1[56]:[^:]*:Silken Puff:7769:/
+233.0 "Fizzling Duster?" #sync / 1[56]:[^:]*:Silken Puff:776A:/
+244.2 "Dust Bluster" sync / 1[56]:[^:]*:Silkie:776C:/
+249.6 "Bracing Suds" sync / 1[56]:[^:]*:Silkie:7757:/
+255.7 "Fresh Puff" sync / 1[56]:[^:]*:Silkie:7766:/
+262.3 "--sync--" sync / 1[56]:[^:]*:Silkie:(7751|7752):/
+263.8 "Squeaky Clean 1" sync / 1[56]:[^:]*:Silkie:7753:/
+265.5 "Squeaky Clean 2" sync / 1[56]:[^:]*:Silkie:7754:/
+266.9 "Puff and Tumble x4" duration 0.7 #sync / 1[56]:[^:]*:Silken Puff:7771:/
+266.9 "Squeaky Clean 3" sync / 1[56]:[^:]*:Silkie:(7755|7756):/
+276.0 "Soaping Spree" sync / 1[56]:[^:]*:Silkie:7767:/
+276.0 "Chilling Duster x4?" #sync / 1[56]:[^:]*:Silken Puff:7768:/
+276.0 "Bracing Duster" #sync / 1[56]:[^:]*:Silkie:7769:/
+# 276.0 "Fizzling Duster?" #sync / 1[56]:[^:]*:Silken Puff:776A:/ # Possible here, but means a mechanic failed
+# 276.0 "Bracing Duster?" sync / 1[56]:[^:]*:Silken Puff:7769:/ # Possible here, but means a mechanic failed
+283.1 "Total Wash" sync / 1[56]:[^:]*:Silkie:7750:/
+292.3 "Bracing Suds / Chilling Suds / Fizzling Suds" sync / 1[56]:[^:]*:Silkie:(7757|7758|7759):/
+294.4 "--sync--" sync / 1[56]:[^:]*:Silkie:79FB:/
+299.4 "Slippery Soap" sync / 1[56]:[^:]*:Silkie:775E:/
+303.5 "Bracing Duster / Chilling Duster / Fizzling Duster" sync / 1[56]:[^:]*:Silkie:(7764|7763|7765):/
+303.8 "Soapsud Static?" #sync / 1[56]:[^:]*:Silken Puff:77ED:/
```

Comment:
Is this a failure as well? If so, then remove?

```suggestion
```

Reply:
Personally, I think given that it's `0.3s` later, you can just drop it from the timeline.  If it was an ability like 3-5s later or something, then it would be worth having.  But, because it's already so close to the existing one, this feels like a "castbar + damage, so don't list the damage" kind of case.